core/src/main/resources/defaults/blueprints/hdp26-data-science-spark2.bp

```diff
@@ -0,0 +1,184 @@
+{
+  "Blueprints": {
+    "blueprint_name": "hdp-data-science-spark2",
+    "stack_name": "HDP",
+    "stack_version": "2.6"
+  },
+  "configurations": [
+    {
+      "core-site": {
+        "fs.trash.interval": "4320"
+      }
+    },
+    {
+      "hdfs-site": {
+        "dfs.namenode.safemode.threshold-pct": "0.99"
+      }
+    },
+    {
+      "hive-site": {
+        "hive.exec.compress.output": "true",
+        "hive.merge.mapfiles": "true",
+        "hive.server2.tez.initialize.default.sessions": "true"
```

Comment:
Missing hiveserver2.mode parameter, it must be set to http, see other blueprints